src/Document/GarbageCollection.php

```diff
@@ -0,0 +1,119 @@
+<?php
+/**
+ * Content
+ *
+ * @package Wp_Graphql_Smart_Cache
+ */
+
+namespace WPGraphQL\SmartCache\Document;
+
+use WPGraphQL\SmartCache\Admin\Settings;
+use WPGraphQL\SmartCache\Document;
+use GraphQL\Server\RequestError;
+
+class GarbageCollection {
+
+	const TAXONOMY_NAME = 'graphql_document_skip_gc';
+	const DISABLED      = 'disabled';
+
+	public function init() {
+		register_taxonomy(
+			self::TAXONOMY_NAME,
+			Document::TYPE_NAME,
+			[
+				'description'        => __( 'Select this option to not clean up this saved GraphQL query document after a number of days. See admin settings.', 'wp-graphql-smart-cache' ),
+				'labels'             => [
+					'name' => __( 'Garbage Collection', 'wp-graphql-smart-cache' ),
+				],
+				'hierarchical'       => false,
+				'public'             => false,
+				'publicly_queryable' => false,
+				'show_admin_column'  => true,
+				'show_in_menu'       => Settings::show_in_admin(),
+				'show_ui'            => Settings::show_in_admin(),
+				'show_in_quick_edit' => false,
+				'meta_box_cb'        => [
+					'WPGraphQL\SmartCache\Admin\Editor',
+					'skip_garbage_collection_input_box_cb',
+				],
+				'show_in_graphql'    => false,
+				// false because we register a field with different name
```

Comment:
Where do we do this?

Reply:
Copy/paste from another with mutations and used register_graphql_field for the Document field we want.